src/Sylius/Behat/Context/Ui/UserContext.php

```diff
@@ -45,21 +46,29 @@
     private $loginPage;
 
     /**
+     * @var RegisterPage
+     */
+    private $registerPage;
+
+    /**
      * @param SharedStorageInterface $sharedStorage
      * @param UserRepositoryInterface $userRepository
      * @param CustomerShowPage $customerShowPage
      * @param LoginPage $loginPage
+     * @param RegisterPage $registerPage
```

Comment:
Since we try to introduce new approach it should be type hinted with an interface. Also check this in spec of this class